core/src/main/java/edu/ucr/cs/riple/core/Config.java

```diff
@@ -458,6 +513,7 @@ public static class Builder {
     public String nullawayLibraryModelLoaderPath;
     public int depth = 1;
 
+    @SuppressWarnings("unchecked")
```

Comment:
Actually this method changed later (at the time of making unit tests), I removed some flags to make configuring easier. For instance we had two separate flags `--nullaway-config-paths` and `scanner-config-paths` and with downstream dependencies analysis activated, a path to a file containing these paths for the rest of the modules. I removed `--nullaway-config-paths` and `scanner-config-paths` flags and now we only require a path to a file which the first entry is for the target module and others are downstream dependencies.

There was always a warning when compiling `core` module and thought since I am modifying this method, add a suppress warning since it's just because of json creation.